src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -292,6 +296,11 @@ def init_partition_fields(partition)
 
       def init_lv_fields(lv)
         @lv_name = lv.basename
+        @stripes = lv.stripes
+        @stripe_size = lv.stripe_size.to_i / DiskSize.KiB(1).to_i
```

Comment:
Does `lv.stripe_size` always come in bytes? Do we need to storage the number of KiB in `@stripe_size`?

Reply:
Yes, according to the documentation, it is always an integer number in KiB (it is not nice, but we want to keep backward compatibility). And when we export this section, we want just the plain number.